spinn_front_end_common/utilities/report_functions/reports.py

```diff
@@ -677,31 +675,29 @@ def generate_comparison_router_report(compressed_routing_tables):
             "Can't open file {} for writing.", file_name)
 
 
-def _search_route(source_placement, key_and_mask):
+def _search_route(source_vertex, key_and_mask):
     """
-    :param ~pacman.model.placements.Placement source_placement:
+    :param ~pacman.model.graphs.machine.MachineVertex source_vertex:
     :param ~pacman.model.routing_info.BaseKeyAndMask key_and_mask:
     :rtype: tuple(str, int)
     """
     # Create text for starting point
     machine = FecDataView.get_machine()
-    source_vertex = source_placement.vertex
     text = ""
     if isinstance(source_vertex, MachineSpiNNakerLinkVertex):
-        text = "        Virtual SpiNNaker Link on {}:{}:{} -> ".format(
-            source_placement.x, source_placement.y, source_placement.p)
+        text = "        Virtual SpiNNaker Link on -> "
```

Comment:
I suspect the fix here is to use x, y, slink in the formatted output to match the placement, as otherwise it isn't too clear where the link comes from

Reply:
Not sure I agree as the slink x, and y are part of the report but added as requested.